src/UtilityTrait.php

```
@@ -0,0 +1,109 @@
+      $field_name = array_shift($row);
+
+      // Assign each value to corresponding entity.
+      // Note: Gherkin's TableNode automatically pads missing cells with empty
+      // strings, so we don't need to validate row length.
+      foreach ($row as $index => $value) {
+        $entities[$index][$field_name] = $value;
+      }
+    }
+
+    return $entities;
+  }
```

PHPMD (2.15.0)

The method utilityTransposeVerticalTable() has a Cyclomatic Complexity of 10. The configured cyclomatic complexity threshold is 10. (undefined)

(CyclomaticComplexity)